lib/Gym/environment.ex

```diff
@@ -25,14 +26,11 @@ defmodule Gyx.Gym.Environment do
     please use #{__MODULE__}.make(ENVIRONMENTNAME)\n")
     HelperAsync.call(python_session, :test, :register_handler, [self()])
 
-    {:ok, %__MODULE__{env: nil,
-                      current_state: nil,
-                      session: python_session,
-                      action_space: nil}}
+    {:ok, %__MODULE__{env: nil, current_state: nil, session: python_session, action_space: nil}}
   end
 
   def start_link(_, opts) do
-    GenServer.start_link(__MODULE__, %__MODULE__{}, opts)
+    GenServer.start_link(__MODULE__, %__MODULE__{action_space: nil}, opts)
```

Comment:
¿estas dos lineas no son equivalentes?

Reply:
`action_space` is now required by `Env` behavior, the compiler asked me to do this change